solr/modules/opentelemetry/src/test/org/apache/solr/opentelemetry/BasicAuthIntegrationTracingTest.java

```diff
@@ -99,5 +104,12 @@ public void testSetupBasicAuth() throws Exception {
             .build();
     req.setBasicAuthCredentials(USER, PASS);
     assertEquals(0, req.process(cloudClient, COLLECTION).getStatus());
+
+    var finishedSpans = getAndClearSpans();
+    assertEquals(1, finishedSpans.size());
+    var span = finishedSpans.get(0);
+    assertEquals("post:/cluster/security/authentication", span.getName());
+    assertEquals("solr", span.getAttributes().get(TraceUtils.TAG_USER));
+    assertEquals(List.of("set-user"), span.getAttributes().get(TraceUtils.TAG_OPS));
```

Comment:
Is it not possible to set the operation name later on with OTEL?  The action/operation is a huge part of what the span *is*; deserves to be a part of the operation name if we can.

Reply:
could you explain a bit better what you expect to see in the operation name? I thought having `post:/cluster/security/authentication` is in line with all other apis.
also one part that is different here is that it's not a single op, there is a list in the body, so I went with a list attribute and the previous approach of taking the first entry to set as a name seems fragile. I am open to suggestions on what we could do.

Reply:
Perhaps `op1,op2,op3:post:/cluster/security/authentication` ?  AFAIK there is no standard here, so we can propose whatever syntax we think is useful.

An alternative would be to leave this be but always create another span at a lower level immediately in front of doing the operation.  At least then there would be a span per operation!

Reply:
re. `op1,op2,op3` I would avoid having the op name dynamic and of unknown length. it makes its usefulness in searching and grouping almost zero.

> An alternative would be to leave this be but always create another span at a lower level immediately in front of doing the operation. At least then there would be a span per operation!

this one makes more sense. will give it a go to see how it looks, but the added complexity of wrapping each operation in a span might be overkill for this part of the code. I see 2 aspects here: performance and auditability and I'm hard pressed to make a good argument for either at this level. but maybe I am missing something.

Reply:
ugh. spoke too soon. you can't break the ops apart, they are passed in as a list to the impl and processed in a loop. so I'd have to update each plugin separately to have one span per op.
as an aside, one thing I realized would be useful here is to have the plugin class that processed the list as an attrib.

Reply:
Let's just keep the list then.  I suppose this is rather specific to this plugin; I am more interested in general/broad cases.

Reply:
I agree with the sentiment. this is a good opportunity to evaluate closely the patterns and make sure things are lean and clean. if we start adding otel data in more and more places, there is a real risk of attributes explosion.